wkcuber/image_readers.py

```diff
@@ -28,6 +28,9 @@ def read_z_slices_per_file(
     ) -> int:
         return 1
 
+    def read_dtype(self, file_name: str) -> str:  # pylint: disable=unused-argument
+        return "uint8"
```

Comment:
why does this default to uint8?

Reply:
This was my workaround because I didn't know how to read the dtype from other libraries. I added a comment that this needs to be done.